src/azul/project/hca/transformers.py

```diff
@@ -14,18 +14,54 @@
 
 def _project_dict(bundle: api.Bundle) -> dict:
     project: api.Project
+    laboratories: List[str]
+    contributors: List[str]
+
     project, *additional_projects = bundle.projects.values()
     reject(additional_projects, "Azul can currently only handle a single project per bundle")
+
+    laboratories = set()
+    contributors = set()
+
+    for contributor in project.contributors:
+        if contributor.laboratory:
+            laboratories.add(contributor.laboratory)
+
+        if contributor.contact_name:
+            contributors.add(contributor.institution)
+
     return {
         'project_title': project.project_title,
-        'project_shortname': project.project_shortname,
-        'laboratory': sorted(list(project.laboratory_names)),
-        'contributors': sorted(list(project.contributors)),
+        'project_shortname': project.project_short_name,
+        'laboratory': sorted(laboratories),
+        'contributors': sorted(contributors),
         'document_id': project.document_id,
+        'contact': _project_contact_dict(project.contributors),
+        'publications': sorted(project.publications),
```

Comment:
Does publication get stored properly? Do you have to turn it into a dict?

Reply:
For the record, yes, it is.